src/d3d12/d3d12_renderer.cpp

```diff
@@ -796,9 +814,62 @@ namespace wr
 							static_cast<D3D12ModelPool*>(model->m_model_pool)->GetIndexStagingBuffer(),
 							0,
 							static_cast<D3D12ModelPool*>(model->m_model_pool)->GetIndexStagingBuffer()->m_size);
+
 						m_bound_model_pool = static_cast<D3D12ModelPool*>(model->m_model_pool);
 						m_bound_model_pool_stride = n_mesh->m_vertex_staging_buffer_stride;
 					}
+
+
+					enum MaterialPBR: unsigned int
+					{
+						PBR_Albedo = 0,
```

Comment:
Enums should be all capitals https://teamwisp.github.io/code_standards/

Reply:
Also a bit of a weird place to define this enum.

Reply:
Done.